app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -383,6 +385,50 @@ class HomeActivityTest {
     }
   }
 
+  @Test
+  fun testHomeActivity_profileNameTextview1_profileNameTextview2_textView1VisibleForAdminNameAndTextView2Gone() {
+    launch<HomeActivity>(createHomeActivityIntent(internalProfileId)).use {
+      onView(
+        atPositionOnView(
+          R.id.home_recycler_view,
+          0,
+          R.id.profile_name_textview
+        )
+      ).check(matches(withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
+        .check(matches(withText("Admin!")))
+      onView(
+        atPositionOnView(
+          R.id.home_recycler_view,
+          0,
+          R.id.profile_name_textview2
+        )
+      ).check(matches(withEffectiveVisibility(ViewMatchers.Visibility.GONE)))
```

Comment:
We cannot test two checks in one go. Test one things at a time. So, it means one check at time. 

```kotlin
onView(
        atPositionOnView(
          R.id.home_recycler_view,
          0,
          R.id.profile_name_textview2
        )
      ).check(matches(withEffectiveVisibility(ViewMatchers.Visibility.GONE)))
```

or 

```kotlin
onView(
        atPositionOnView(
          R.id.home_recycler_view,
          0,
          R.id.profile_name_textview2
        )
      ).check(matches(withText("Admin!")))
```

Also, for visibility, follow my previous comments.

Reply:
Thanks @anandwana001